Flaky-test: FilterEntryTest. testFilter

### Search before asking

- [X] I searched in the [issues](https://github.com/apache/pulsar/issues) and found nothing similar.


### Example failure

https://github.com/apache/pulsar/actions/runs/7176247224/job/19541021647?pr=21711
https://github.com/apache/pulsar/actions/runs/7208049048/job/19637548110?pr=21704

### Exception stacktrace

<!-- copy-paste the stack trace in the code block below -->
```
Error:  Tests run: 20, Failures: 1, Errors: 0, Skipped: 16, Time elapsed: 24.003 s <<< FAILURE! - in org.apache.pulsar.broker.service.plugin.FilterEntryTest
  Error:  org.apache.pulsar.broker.service.plugin.FilterEntryTest.testFilter  Time elapsed: 1.917 s  <<< FAILURE!
  java.lang.AssertionError: expected:<10> but was:<8>
  	at org.testng.AssertJUnit.fail(AssertJUnit.java:65)
  	at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:467)
  	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:88)
  	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:318)
  	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:328)
  	at org.apache.pulsar.broker.service.plugin.FilterEntryTest.testFilter(FilterEntryTest.java:267)
  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
  	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
  	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
  	at org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
  	at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
  	at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
  	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
  	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
  	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
  	at java.base/java.lang.Thread.run(Thread.java:840)
```


### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

--- a/pulsar-broker/src/test/java/org/apache/pulsar/broker/service/plugin/FilterEntryTest.java
+++ b/pulsar-broker/src/test/java/org/apache/pulsar/broker/service/plugin/FilterEntryTest.java
@@ -255,7 +255,7 @@ public class FilterEntryTest extends BrokerTestBase {
 
         int counter = 0;
         while (true) {
-            Message<String> message = consumer.receive(1, TimeUnit.SECONDS);
+            Message<String> message = consumer.receive(5, TimeUnit.SECONDS);
             if (message != null) {
                 counter++;
                 consumer.acknowledge(message);
@@ -289,7 +289,7 @@ public class FilterEntryTest extends BrokerTestBase {
 
         counter = 0;
         while (true) {
-            Message<String> message = consumer.receive(1, TimeUnit.SECONDS);
+            Message<String> message = consumer.receive(5, TimeUnit.SECONDS);
             if (message != null) {
                 counter++;
                 consumer.acknowledge(message);
@@ -321,7 +321,7 @@ public class FilterEntryTest extends BrokerTestBase {
         }
         counter = 0;
         while (true) {
-            Message<String> message = consumer.receive(1, TimeUnit.SECONDS);
+            Message<String> message = consumer.receive(5, TimeUnit.SECONDS);
             if (message != null) {
                 counter++;
                 consumer.acknowledge(message);

